### ISSUE TITLE
dd-sdk-android-timber requires dd-sdk-android:1.2.1

### ISSUE THREAD

**Issue by mduong** [DataDog/dd-sdk-android#138]
Thanks for taking the time for reporting an issue!

**Describe what happened**
Include any error message or stack trace if available.

In trying to include `dd-sdk-android-timber` as a dependency, resolution fails because it cannot find `dd-sdk-android:1.2.1`. Looking at the POM file for `dd-sdk-android-timber`, it appears that it requires 1.2.1 as a dependency (https://dl.bintray.com/datadog/datadog-maven/com/datadoghq/dd-sdk-android-timber/), but that does not exist. I believe it should be 1.2.0.

Error message:
```
ERROR: Failed to resolve: dd-sdk-android:dd-sdk-android:1.2.1
```

**Steps to reproduce the issue:**
Add this to your build.gradle and perform a Gradle sync:
```
repositories {
    maven { url “https://dl.bintray.com/datadog/datadog-maven" }
}

dependencies {
    implementation "com.datadoghq:dd-sdk-android:1.2.0"
    implementation "com.datadoghq:dd-sdk-android-timber:1.2.0"
}
```

**Describe what you expected:**
Expected Gradle dependency resolution to complete successfully.

**Additional context**
 - Android OS version:
 - Device Model:
 - Datadog SDK version:
 - Versions of any other relevant dependencies (OkHttp, …): 
 - Proguard configuration: 
 - Gradle Plugins:

**Comment by xgouchet** [comment]
Indeed there's an issue in the published pom. We'll check that ASAP.

**Comment by xgouchet** [comment]
Thanks again for reporting this issue @mduong. It's now fixed : 
 - We published a version `1.2.1` of the libraries (both `dd-sdk-android` and `dd-sdk-android-timber`);
- We fixed our publishing script so that this issue doesn't happen anymore :)

**Comment by mduong** [comment]
Thanks @xgouchet. I think there may still be an issue. Upon further inspection of the `dd-sdk-android-timber` POM, I believe the groupId for dd-sdk-android should be com.datadoghq instead of dd-sdk-android.

**Comment by alparp27** [comment]
I am also having the same issue.

**Comment by rafaela-stockx** [comment]
Currently trying to use the latest version and seeing the same behavior. 
`"com.datadoghq:dd-sdk-android:1.2.1"` 
`"com.datadoghq:dd-sdk-android-timber:1.2.1"` 

**NOTE** : The `1.2.1` version gradle dependency work as expected if there is no need to pull the timber integration version.

**Comment by xgouchet** [comment]
Indeed there's was another issue, nicely spotted @rafaela-stockx 
Should really be fixed now.

**Comment by mduong** [comment]
Confirming fixed in 1.2.2.
